### DISCUSSION TITLE
Row And Column in Firefox

### DISCUSSION THREAD

**Discussion by aseph8083** [twbs/bootstrap#36152] [Q&A]
i'm using bootstrap 5, i display 2 row on chrome display as expected, but on firefox it doesn't fit, how can i solve this so this view can match?

this is what it looks like when using chrome
![image](https://user-images.githubusercontent.com/87680962/162901079-c1e2a0d0-1ea9-42f9-afdd-f0babfffc902.png)

and this when using firefox
![boots](https://user-images.githubusercontent.com/87680962/162904623-317dcf40-4152-4651-9a06-b46719945aa6.png)

**Comment by WinterSilence** [comment]
Very nice pictures! You can solve your problem like this:
![изображение](https://user-images.githubusercontent.com/3521094/163040697-20e2f780-109c-446f-a372-1bd8a72aa6e2.png)
